### PR TITLE
Completed css-flexbox

### PR DESCRIPTION
**Pull request by julieluangpraxay**
Screenshot 2023-07-20 at 2 40 51 PM

### CODE REVIEW

File: css-flexbox/justify-content.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    <link rel="stylesheet" href="pokemon.css" />
+    <style>
+      /* your styles here 👇 */
+      #card-deck {
+        justify-content: center;
+        display: flex;
+      }
+      .card-wrapper {
+        flex-basis: 25%;
+        justify-content: center;
```
Comment (shawnkost) [inline css-flexbox/justify-content.html:18]: `justify-content` is a parent property, which means it gets applied where you also apply `display: flex`. So this isn't doing anything, so it should be removed

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change here!

File: css-flexbox/justify-content.html (hunk 1 above)
Comment (julieluangpraxay) [reply css-flexbox/justify-content.html:18]: Screenshot 2023-07-22 at 11 07 49 AM

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**